k8s/openebs-operator.yaml

```diff
@@ -512,8 +515,7 @@ spec:
           exec:
             command:
             - pgrep
-            - -f
-            - ".*ndm"
+            - "ndm"
```

Comment:
@kmova @akhilerm just using `ndm` to grep here as its only 3 char cmd name.

Reply:
yes. That will work

Reply:
how about we keep it consistent with other liveness checks.